{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/config/settings/base.py

```diff
@@ -85,9 +81,9 @@
     "hijack.contrib.admin",
 ]
 LOCAL_APPS = [
-    "{{cookiecutter.repo_name}}.home",
-    "{{cookiecutter.repo_name}}.user",
-    "{{cookiecutter.repo_name}}.util",
+    "{{cookiecutter.repo_name}}.home.apps.HomeConfig",
+    "{{cookiecutter.repo_name}}.user.apps.UserConfig",
+    "{{cookiecutter.repo_name}}.util.apps.UtilConfig",
```

Comment:
Not strictly necessary. I think this used to be required to get anything in the AppConfig.ready to run - since we were using default_app_config (though missing from user app) that wasn't necessary. I think the new default is whatever is in there if there's only one AppConfig class - but nicer to be explicit IMO.